Resources/Migrations/LostParadiseMigrations.yml

```
@@ -61,3 +61,8 @@ FluffyMango: LPPFluffyMango
 FluffyStan: LPPFluffyStan
 FluffyWatermelon: LPPFluffyWatermelon
 RingBox: LPPRingBox
+LPPCandy2: LPPCandy
+LPPCandy3: LPPCandy
+LPPCandy4: LPPCandy
+HalloweenTree01: LPPHalloweenTree01
+HalloweenTree02: LPPHalloweenTree02
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)